src/witan/app/handler.clj

```diff
@@ -150,24 +150,14 @@
                                              version :- java.lang.Long]
                                :summary "Returns a forecast of the specified id and version"
                                (forecast/forecast {:id id :version version}))
-                   (sweet/POST* "/forecasts/:id" []
-                                :summary "Creates a new version of this forecast with the specified updated"
-                                (not-implemented))
+                   (sweet/POST* "/forecasts/:id/versions" {:as request}
+                                :path-params [id :- java.util.UUID]
+                                :summary "Creates a new version of this forecast with the specified updates and run it"
+                                (forecast/version {:id id :user-id (:identity request)}))
                    (sweet/GET* "/forecasts/:id/:version/output/:type" []
                                :summary "Downloads an output of the given type"
                                (not-implemented))
-                   (sweet/POST* "/forecasts/:id/:version/input/:category" {:as request}
```

Comment:
Ok excellent, I remember now, and we talked about a background job to erase s3 files that didn't have certain metadata (indicating that they'd been uploaded but then subsequently never used), is that right?